docs/usage/self-hosted-configuration.md

```diff
@@ -130,6 +130,16 @@ e.g.
 
 This configuration will be applied after all other environment variables so that it can be used to override defaults.
 
+## detectGlobalManagerConfig
+
+The purpose of this capability is to allow a bot admin to configure manager-specific files such as a global `.npmrc` file, instead of configuring it in Renovate config.
+
+This feature is disabled by default in the CLI because it may prove surprising or undesirable for some users who don't expect Renovate to go into their home directory and import registry or credential information.
+However, it is enabled by default in the official Renovate Docker images because if anyone chooses to add such config files to them then it's clearly not unintentional.
```

Comment:
This can be tricky. We rebuild images for multiple versions, so we get invalid configs for older versions?

I think we should defer this doc's until this feature is fully landed?

🤔🤔🤔

Reply:
In general our docs represent the current version's capabilities, so in that context wouldn't it still be correct?

Reply:
Sure, but I think the enabled by default in official docker images will be tricky. As we should only add that setting for supported versions, otherwise we get config errors.

So the tricky part is to auto enable it for our docker images only.

Reply:
if we set it in env, should be ignored by older versions?

Reply:
anyway I can delay the docs for now